src/atomistics/calculators/lammps/libcalculator.py

```
@@ -470,8 +470,8 @@ def calc_molecular_dynamics_thermal_expansion_with_lammpslib(
     seed: int = 4928459,
     dist: str = "gaussian",
     couple_xyz: bool = False,
-    lmp=None,
-    output_keys=OutputThermalExpansion.keys(),
+    lmp: Optional[LammpsASELibrary] = None,
+    output_keys: list[str] = OutputThermalExpansion.keys(),
```

Ruff (0.15.14)

[warning] Do not perform function call `OutputThermalExpansion.keys` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)